### PR TITLE
Adding linkage monitor Kokoro config

### LINKED ISSUE #5518: Apply Linkage Monitor at presubmit build
CC @chingor13 . I'll prepare Kokoro build config and job config for review.

Linkage Monitor started working in https://github.com/googleapis/google-http-java-client. Let's apply this to google-cloud-java

 **Is your feature request related to a problem? Please describe.**
feature request for continuous build.
 **Describe the solution you'd like**
apply linkage monitor  with Google Libraries BOM
https://github.com/GoogleCloudPlatform/cloud-opensource-java/tree/master/linkage-monitor
 **Describe alternatives you've considered**
N/A
 **Additional context**
Linkage Monitor is a tool to identify inconsistency between libraries as part of presubmit phase.
https://github.com/GoogleCloudPlatform/cloud-opensource-java/tree/master/linkage-monitor

### PR DESCRIPTION
**Pull request by suztomo**
Fixes #5518 

@chingor13 Piper change will come soon.

### CODE REVIEW

File: .kokoro/linkage-monitor.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+cd github/google-cloud-java/
+
+# Print out Java version
+java -version
+echo $JOB_TYPE
+
+# Linkage Monitor uses SNAPSHOT version but google-cloud-java's HEAD does not
+# have SNAPSHOT
+mvn versions:set -DnextSnapshot=true
```
Comment (suztomo) [inline .kokoro/linkage-monitor.sh]: Workaround of missing SNAPSHOT version problem.

**Comment by suztomo** [comment]
After https://github.com/googleapis/google-cloud-java/pull/5530 is merged, I'll run test again.

**Comment by suztomo** [comment]
"Merging master (351bee0)" was not included in the last Kokoro build (#4). Would you kick off Kokoro again?

**Comment by chingor13** [comment]
Kokoro runs "as if merged", but we can retrigger again.

**Comment by suztomo** [comment]
Found expected message. This PR is ready for review and merge.

>Snapshot versions have the same 106 errors as baseline

File: .kokoro/linkage-monitor.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2019 Google Inc.
```
Comment (chingor13) [inline .kokoro/linkage-monitor.sh]: nit: this should be Google LLC

Comment (suztomo) [reply .kokoro/linkage-monitor.sh]: Updated

**Comment by suztomo** [comment]
@chingor13 Would you merge this? (I don't have write access)

**Pull request merged**